app/api-v1/controllers/Recipe/index.js

```diff
@@ -51,7 +66,7 @@ module.exports = {
         ],
       }
       const token = await runProcess(payload, req.token)
-      const transaction = await client
+      const transaction = await db.client
```

Comment:
the response doesn't match any in the API spec such as https://github.com/digicatapult/inteli-api/blob/a96a90a32fb17ce342cac4becaf787b3ec625d62/app/api-v1/api-doc.js#L306

Reply:
updated